src/components/ClaimForm.js

```diff
@@ -356,12 +368,14 @@ class ClaimForm extends Component {
       }
       if (!items.length && !services.length) return !!this.canSaveClaimWithoutServiceNorItem;
     }
-
-    if (
```

Comment:
it seems that you are removing the patient condition check, is it on purpose ? 
poke: @dragos-dobre